### DISCUSSION TITLE
Top Of History Page Starting From Oldest

### DISCUSSION THREAD

**Discussion by ghost** [FreeTubeApp/FreeTube#5239] [Q&A]
I just imported my history from youtube and its showing the oldest date watched instead of the newest. how can i change to show the recently watched on the top?
![Screenshot from 2024-06-08 22-06-35](https://github.com/FreeTubeApp/FreeTube/assets/150875575/79ab29e9-9135-4880-a523-2ba64fa23977)

**Comment by absidue** [comment]
It's a known issue, you can just restart the app to solve it: https://github.com/FreeTubeApp/FreeTube/issues/5174#issuecomment-2132219607
